src/base/BaseActionsRouter.sol

```diff
@@ -11,6 +11,10 @@ import {Actions} from "../libraries/Actions.sol";
 abstract contract BaseActionsRouter is SafeCallback {
     using CalldataDecoder for bytes;
 
+    /// @notice used to signal that the recipient of an action should be the _msgSender of address(this)
+    address internal constant MSG_SENDER = address(1);
```

Comment:
can we just have a shared constants file with all of these?

msg sender, address this, contract balance, open delta?